.github/workflows/release.yml

```diff
@@ -8,26 +8,6 @@ env:
   PYTHON_VERSION: "3.9"
 
 jobs:
-  lint:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v4
-      - name: Set up Python
-        uses: actions/setup-python@v5
-        with:
-          python-version: "3.9"
-
-      # The installation is required as mypy must be run in the local system environment, not in the pre-commit environment.
-      - name: Install required dependencies
-        run: |
-          python3 -m pip install --upgrade pip poetry
-          poetry config virtualenvs.create false
-          poetry install
-
-      - name: Linting and static code checks
-        run: |
-          pre-commit run --all-files
-
```

Comment:
**question**: Not needed here because we do it in build.yml?

Reply:
Yea, no need to re-run pre-commit on merge to main given the branch protections and strict requirements for passing on PRs toward main